src/Download.vue

```diff
@@ -33,33 +33,70 @@
   </div>
 </template>
 
-<style scoped>
+<script>
+export default {
+  data: function() {
+    return {
+      windows: "",
+      linux: "",
+      loaded: false
+    };
+  },
+  created: function() {
+    this.getLatestDownloadLinks();
+  },
+  methods: {
+    getLatestDownloadLinks: function() {
+      return this.$http
+        .get(`https://api.github.com/repos/inexorgame/inexor-core/releases`)
+        .then(
+          response => {
+            const { tag_name } = response.body[0];
+            this.windows = `https://github.com/inexorgame/inexor-core/releases/download/${tag_name}/Inexor-${tag_name}-win64.zip`;
+            this.linux = `https://github.com/inexorgame/inexor-core/releases/download/${tag_name}/Inexor-${tag_name}-Linux.zip`;
+            this.loaded = true;
+          },
+          response => {
+            alert("something went wrong");
```

Comment:
Instead of throwing an alert at the average user, it would be better to just link without any version and platform strings to https://github.com/inexorgame/inexor-core/releases/